app_feup/lib/redux/action_creators.dart

```diff
@@ -222,6 +222,10 @@ ThunkAction<AppState> getUserExams(Completer<Null> action,
       store.dispatch(SetExamsStatusAction(RequestStatus.busy));
 
       final List<Exam> exams = await extractExams(store, parserExams);
+      
+      exams.sort((exam1, exam2) {
+        return exam1.date.compareTo(exam2.date);
+      });
```

Comment:
If I wanted to nitpick, I'd change to an arrow function like `exams.sort((exam1, exam2) => exam1.date.compareTo(exam2.date))` ;P